nih_plug_iced/src/editor.rs

```diff
@@ -96,8 +99,8 @@ impl<E: IcedEditor> Editor for IcedEditorWrapper<E> {
                     // FIXME: Rust analyzer always thinks baseview/opengl is enabled even if we
                     //        don't explicitly enable it, so you'd get a compile error if this line
                     //        is missing
-                    #[cfg(not(feature = "opengl"))]
-                    gl_config: None,
+                    // #[cfg(not(feature = "opengl"))]
+                    // gl_config: None,
```

Comment:
If this is not needed anymore then the whole section including the comment can be removed.